x-pack/plugins/ml/public/application/memory_usage/nodes_overview/allocated_models.tsx

```diff
@@ -106,7 +106,8 @@ export const AllocatedModels: FC<AllocatedModelsProps> = ({
       name: (
         <EuiToolTip
           content={i18n.translate('xpack.ml.trainedModels.nodesList.modelsList.allocationTooltip', {
-            defaultMessage: 'number_of_allocations times threads_per_allocation',
+            defaultMessage:
+              'Number of allocations per node multiplied by number of threads per allocation',
```

Comment:
Currently we show the same tooltip for stateful and serverless, and on serverless 'nodes' sort of don't exist. Although it is less clear, I'd therefore suggest

```
Number of allocations multiplied by number of threads per allocation
``` 

until we can add an extra check in for serverless (note there is some WIP here - https://github.com/elastic/kibana/pull/191960).

Reply:
ah well the request that sparked this change was to clarify whether it was indeed per node, so your edit doesn't actually change much 🤔

Reply:
we still return stuff about shards in basic es apis on serverless though currently

Reply:
This message should depend on whether we can show node information.
We have a `showNodeInfo` flag
`const { showNodeInfo } = useEnabledFeatures();`
It'll be `false` in serverless.

I'd suggest something like:
```
          content={
            showNodeInfo
              ? i18n.translate(
                  'xpack.ml.trainedModels.nodesList.modelsList.allocationTooltipNodes',
                  {
                    defaultMessage:
                      'Number of allocations per node multiplied by number of threads per allocation',
                  }
                )
              : i18n.translate('xpack.ml.trainedModels.nodesList.modelsList.allocationTooltip', {
                  defaultMessage:
                    'Number of allocations multiplied by number of threads per allocation',
                })
          }
```

Reply:
Thanks! Had a crack at implementing that in https://github.com/elastic/kibana/pull/197099/commits/8530fbd3365fd89325aee8108f5419c8e24843e2